src/helpers/get-survey-answers.ts

```diff
@@ -0,0 +1,33 @@
+import MyDataHelps, { Guid, SurveyAnswer, SurveyAnswersPage, SurveyAnswersQuery } from "@careevolution/mydatahelps-js";
+import {parseISO} from "date-fns";
+
+export default async function(props : SurveyAnswersQuery) : Promise<SurveyAnswer[]> {
+
+    async function getSurveyAnswers(): Promise<SurveyAnswer[]> {
+        let dataPage = await getSurveyDataPage();
+        let allData = dataPage.surveyAnswers;
+        while (dataPage.nextPageID) {
+            dataPage = await getSurveyDataPage(dataPage.nextPageID);
+            allData = allData.concat(dataPage.surveyAnswers);
+        }
+        return allData;
+    }
+
+    async function getSurveyDataPage(pageID?: Guid): Promise<SurveyAnswersPage> {
+        var queryParameters: SurveyAnswersQuery = props;
+        if (pageID) {
+            queryParameters.pageID = pageID;
+        }
+
+        return MyDataHelps.querySurveyAnswers(queryParameters);
+    }
+
+    const results = await getSurveyAnswers();
+    var sortedResults = (results).sort((a, b) => {
+            if (parseISO(a.date) > parseISO(b.date)) { return -1; }
```

Comment:
Sorting should be unnecessary; answers are already sorted by date via the API...it also seems out of place in this function